e2e/data/expected_context.yaml

```
@@ -16,8 +16,30 @@
 # under the License.
 #
 segmentItems:
-- segmentSize: 1
+- segmentSize: 2
```

```suggestion
- segmentSize: gt 2
```

I think a certain number is not safe for test stability. Because the e2e tool could re-try if the first validation doesn't pass.